networking/kvm.go

```diff
@@ -322,6 +389,18 @@ func (n *Networking) teardownKvmNets() {
 		case "ptp", "bridge":
 			// remove tuntap interface
 			tuntap.RemovePersistentIface(an.runtime.IfName, tuntap.Tap)
+
+		case "macvlan":
+			link, err := netlink.LinkByName(an.runtime.IfName)
+			if err != nil {
+				log.Printf("Cannot find link `%v`: %v", an.runtime.IfName, err)
```

Comment:
Does it make sense to proceed with the teardown after this, or should we just `continue`?